### DISCUSSION TITLE
XArray issue of reading groups

### DISCUSSION THREAD

**Discussion by aninramesh** [ReedEspinosa/GSFC-Retrieval-Simulators#3] [Q&A]
Hi @NirandiJ ,

Apparently, Xarray has some issues reading groups in a .nc file. You can explore more here https://github.com/pydata/xarray/issues/2916 

I have used Xarray with ABI data which doesn't have groups. 

Anin

**Comment by NirandiJ** [comment]
Thanks I will look into this.

**Comment by NirandiJ** [comment]
@aninramesh I think I figured out the issue with the imshow. I need to re-grid the data into the extend grid and map image data first to use the imshow. The same way we plot MODIS data in Basemap. Will keep you posted.
